phase1/openstack/README.md

```diff
@@ -0,0 +1,45 @@
+# Getting started on OpenStack
+
+Prerequisites:
+* docker-engine which can be downloaded [here](https://docs.docker.com/engine/installation/)
```

Comment:
Nit: Do we need to specify docker-engine specifically? Can we instead point to the docker install docs in general in case someone is using Linux and doesn't need the whole suite?

Reply:
s/docker-engine/docker